tei/rjmw.0.xml

```diff
@@ -3822,8 +3822,10 @@
               <sp who="#sir_william">
                 <speaker>SIR WILLIAM.</speaker>
                 <p> Wenn sie Wunder tun können, so laß sie herein kommen! – Laß mich nicht länger, Waitwell, bei diesem tötenden Anblicke verweilen. Ein Grab soll beide umschließen. Komm, schleunige Anstalt zu machen, und dann laß uns auf Arabellen denken. Sie sei, wer sie sei: sie ist ein Vermächtnis meiner Tochter. <hi>Sie gehen ab, und das Theater fällt zu.</hi> </p>
+			  </sp>

```

Comment:
Hier sind noch irgendwelche tabs in den Zeilen mit `</sp>` und `</trailer>`

Reply:
Alles klar, ich werde das korrigieren.